spec/components/personas/view_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+module Personas
+  describe View do
+    alias_method :component, :page
+
+    let(:persona_id) { 1 }
+    let(:persona) { build(:user, id: persona_id, provider: build(:provider)) }
+
+    before do
+      render_inline(described_class.new(persona: persona))
+    end
+
+    it "renders persona's name" do
+      expect(component).to have_text(persona.name)
```

Comment:
I'm aware that other view component specs attempt to find text within a specified tag. I'm not wholly against the idea but not a fan of it either. HTML been the presentation layer can change quite often but the app behaviour remains the same so I don't see much value in checking for tags with a class or ID value. With that said, I'm open to changing my view on this.